src/modules/services_manager_service.rb

```diff
@@ -358,10 +358,15 @@ def error_messages_for(service)
     #
     # @return [String] Error message
     def active_error_message_for(service)
-      change = service.active? ? 'start' : 'stop'
-      status = service.running? ? 'running' : 'not running'
-      _("Could not %{change} %{service} which is currently %{status}." %
-        { change: change, service: service.name, status: status })
+      change = service.active? ? N_("start") : N_("stop")
```

Comment:
Why `N_`?

Reply:
Those strings will be used inside another translation, see https://github.com/yast/yast-services-manager/pull/163/files/5a32955d5c401087795aaf742aca270c5f1c002d#diff-c00d8153684ebbbf21892421501eb127R365

I guess the string should be marked for translation?

Reply:
Tecnhically, the `N_` will only mark the string for translation (it is useful on class methods, constants declarations, etc. where the `_` method does not work) but it won't translate the string. So in this case, you should use `_` instead.

However, do we need to translate these strings? I mean: are they shown in the UI or only in the logs? (I know they were translated in the past). If this message is only written in the logs, we should skip the translation.

Reply:
Otherwise, if it is shown in the UI, we should add some comment clarifying how the final string is built to make it easier for our translators.